flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-mysql/src/test/java/org/apache/flink/cdc/connectors/mysql/source/MySqlFullTypesITCase.java

```diff
@@ -412,7 +412,7 @@ private void testCommonDataTypes(UniqueDatabase database) throws Exception {
 
         expectedSnapshot[30] = null;
         // The json string from binlog will remove useless space
-        expectedSnapshot[44] = BinaryStringData.fromString("{\"key1\":\"value1\"}");
+        expectedSnapshot[44] = BinaryStringData.fromString("{\"key1\": \"value1\"}");
```

Comment:
This additional assignment was meant to fix the format discrepancy and could be removed now.

Reply:
Resolved